### ISSUE TITLE
flatpak permission-set flatpak updates any no

### ISSUE THREAD

**Issue by damianatorrpm** [flatpak/flatpak#3345]
A better syntax to users who wish to disable self updates for all applications.
`flatpak permission-set flatpak updates $APPID no`
sets this only for one application. I propose
`flatpak permission-set flatpak updates any no`
to disable it by default for everything (even what gets installed after setting above)

**Comment by matthiasclasen** [comment]
The flatpak permission commands are not really meant as a convenient end-user  UI.
They are more like tweak/debug tools.

If controlling self-updates becomes important one day, there should probably be control-center
UI for it. But so far, there are 0 applications doing this, so it is not exactly urgent.

**Comment by damianatorrpm** [comment]
> But so far, there are 0 applications doing this, so it is not exactly urgent.

Yes, the report should keep that suggestion in mind of the developers for future design considerations in that area.
